dtclient/client.go

```diff
@@ -3,11 +3,11 @@ package dtclient
 import (
 	"crypto/tls"
 	"crypto/x509"
-	"errors"
 	"net/http"
 	"net/url"
 	"strings"
 
+	"github.com/pkg/errors"
```

Comment:
Do we need the import here?

Reply:
yes, there are two "errors" packages, "errors" and "pkg/errors". Since "errors", the package previously imported here, also includes a `.New` method, the lines where it was used are not shown in the changes.
E.g. see line 117 `return nil, errors.New("tokens are empty")`.

"errors" does not include stacktraces afaik, but "pkg/errors" does.